### DISCUSSION TITLE
How to delete all emails from Postal server?

### DISCUSSION THREAD

**Discussion by mdobydullah** [postalserver/postal#2299] [Help with using Postal]
Hi,

I want to delete all emails. Is there any way to delete all queued & sent emails?

![Screenshot_2](https://user-images.githubusercontent.com/13184472/208849412-e909a2a8-d311-4c4d-bf8d-9036ed6c6785.png)

Thanks in advance.

**Comment by willpower232** [comment]
You would have to do this directly in the database I believe.

**Reply by mdobydullah** [reply]
@willpower232 Thanks for your reply. Then I'll delete from the database.
